userguide/content/en/docs/Getting started/_index.md

```diff
@@ -201,6 +201,7 @@ To clone Docsy into your project's `theme` folder, run the following commands fr
 ```
 cd themes
 git clone https://github.com/google/docsy
+git submodule update --init --recursive
```

Comment:
Good call, I know we document this elsewhere but users might try running Hugo (as you did!) before they see it.

Might just expand on it a bit so they know why they're doing it, so:

```
cd themes
git clone https://github.com/google/docsy
```
If you want to build and/or [serve your site locally](/docs/deployment/#serving-your-site-locally), you also need to get local copies of the theme’s own submodules:

```
git submodule update --init --recursive
```

Reply:
I have added it